sdk/core/Azure.Core/src/DelayStrategy.cs

```diff
@@ -29,10 +26,11 @@ public abstract class DelayStrategy
         /// delay used will be a random double between 0.8 and 1.2. If set to 0, no jitter will be applied.</param>
         protected DelayStrategy(TimeSpan? maxDelay = default, double jitterFactor = DefaultJitterFactor)
         {
+            _minJitterFactor = 1.0 - jitterFactor;
```

Comment:
Do we care if this goes negative?

Reply:
This should be equivalent to what's there, I think?

Reply:
If someone passes in `jitterFactor` of -200, then you're likely to get in a state where min is greater than max.    Are we ok with that or should we validate/cap jitterFactor at a min of 0?

Reply:
I'm trying _not_ to change the behavior of this type in this PR, though.  If you feel strongly that the behavior of the type should change, let's file an issue to track the work.

Reply:
I don't feel strongly, just calling out what looks like a gap.   I trust your judgement on whether we think it's a problem or are willing to allow it.

Reply:
Since it's a fairly advanced type, I think it's ok to leave the behavior we have in Azure.Core today -- if we find that users calling these APIs have trouble with them, we can revisit the issue at that time.